diff -Naurp a/setup.py b/setup.py
--- a/setup.py	2016-07-06 21:16:24.000000000 -0500
+++ b/setup.py	2017-02-23 16:05:18.939736495 -0600
@@ -616,8 +616,8 @@ setup(name="psycopg2",
       long_description="\n".join(readme.split("\n")[2:]).lstrip(),
       classifiers=[x for x in classifiers.split("\n") if x],
       data_files=data_files,
-      package_dir={'psycopg2': 'lib', 'psycopg2.tests': 'tests'},
-      packages=['psycopg2', 'psycopg2.tests'],
+      package_dir={'psycopg2': 'lib'},
+      packages=['psycopg2'],
       cmdclass={
           'build_ext': psycopg_build_ext,
           'build_py': build_py, },
